src/CodeGeneration.Roslyn.Tool/Program.cs

```diff
@@ -16,8 +16,10 @@ static int Main(string[] args)
             string generatedCompileItemFile = null;
             string outputDirectory = null;
             string projectDir = null;
+            bool version = false;
             ArgumentSyntax.Parse(args, syntax =>
             {
+                syntax.DefineOption("version", ref version, "Show version of this tool.");
```

Comment:
"...of this tool (and exits)."

That way it's clear it's not really an option, but a command that ignores anything else on the command line.

Reply:
Good point.